frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import * as plugins from '../../../../../../../public/plugins';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../../../../../../public/components/dashboard/dashboard-card';
+import { CapacityBody, CapacityItem } from '../../../../../../../public/components/dashboard/capacity-card';
+import { withDashboardResources, DashboardItemProps } from '../../../../../../../public/components/dashboards-page/with-dashboard-resources';
+import { humanizeBinaryBytesWithoutB } from '../../../../../../../public/components/utils';
+import { getInstantVectorStats, getRangeVectorStats, GetStats } from '../../../../../../../public/components/graphs/utils';
+import { CAPACITY_CARD_QUERIES } from '../../../../constants/storage';
+
+const defaultQueries = {
```

Comment:
@aruniiird please move these queries to `constant/storage.tsx`. How I did it in this PR.

https://github.com/openshift/console/pull/1803/files#diff-dd57dd229b99291aa21aa56a53d95b00

Reply:
As suggested in your PR, I have moved the variables into queries.ts, which helps to place labels and queries in a single file.